### PR TITLE
fix: internal message bugs

### LINKED ISSUE #635: Implicitmessage task report unknown CID error
### Describe the bug:
Implicitmessage task report the following error

`visor/task/messagesexecutions   messageexecutions/message.go:65 --cannot derive actor name from unknown CID: bafkqaddgnfwc6mzpojsxoylsmq (maybe we need up update deps?)`

### Steps to Reproduce:

./visor walk  --tasks=implicitmessage --from=1011440

### Debug info:

https://github.com/filecoin-project/sentinel-visor/blob/da50470fde61bc1a99b448260235e3c79fdd0d73/lens/util/repo.go#L466

function ActorNameByCode return "unknown"

https://github.com/filecoin-project/sentinel-visor/blob/da50470fde61bc1a99b448260235e3c79fdd0d73/lens/util/repo.go#L35

Found out that imported `builtin` package is v5.

github.com/filecoin-project/sentinel-visor/chain/actors/builtin maybe the right package

### LINKED ISSUE #636: Implicitmessage task report error
### Describe the bug:
https://github.com/filecoin-project/sentinel-visor/blob/da50470fde61bc1a99b448260235e3c79fdd0d73/tasks/messageexecutions/message.go#L158

Need check `len(errorsDetected) > 0` as "message" task do

### PR DESCRIPTION
**Pull request by frrist**
- fixes #635
- fixes #636

### CODE REVIEW

File: lens/lily/impl.go [hunk 1]
```diff
@@ -178,21 +178,22 @@ func (m *LilyNodeAPI) GetMessageExecutionsForTipSet(ctx context.Context, ts *typ
...
-		if err != nil {
-			return nil, xerrors.Errorf("failed to compute execution trace for tipset: %s", pts.Key().String())
-		}
-		// the above call will populate the msgMonitor with an execution trace for this tipset, get it.
-		executions, err = msgMonitor.ExecutionFor(pts)
-		if err != nil {
-			return nil, xerrors.Errorf("failed to find execution trace for tipset: %s", pts.Key().String())
-		}
-	}
 	if err != nil {
```
Comment (iand) [inline lens/lily/impl.go:181]: Can you describe the fix here because it's not obvious?

Comment (iand) [reply lens/lily/impl.go:181]: The fix works but what was the bug? Shadowing of err?

Comment (frrist) [reply lens/lily/impl.go:181]: Yeah, basically that. After the call to
```go
executions, err := msgMonitor.ExecutionFor(pts)
	if err == modules.ExecutionTraceNotFound {
```

`err` is still non-nill at the error check on line 181, even though it had been resolved in the `err == modules.ExecutionTraceNotFound` check.

File: lens/util/repo.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/filecoin-project/lotus/api"
 	"github.com/filecoin-project/lotus/blockstore"
 	"github.com/filecoin-project/lotus/build"
+	builtin "github.com/filecoin-project/lotus/chain/actors/builtin"
```
Comment (frrist) [inline lens/util/repo.go:17]: This was the other fix. Lotus maintains this package to work with all versions of actor codes.

**Review by iand — APPROVED** [review]
I don't

**Pull request merged**